windows/src/Makefile

```diff
@@ -186,6 +186,39 @@ test-uiaccess:
     $(MAKE) test-uiaccess
     cd $(ROOT)\src
 
+#
+# Build Keyman projects for coverity; call this with
+#   make test-coverity
+# * Set environment variable COVBUILD to path to cov-build.exe
```

Comment:
Would developers ever need to install coverity locally? If so, can you include notes in windows/src/README.md?

Reply:
Yep. I will be doing coverity as a separate PR once I've sorted out how it all works. This is groundwork for that.